core/common/src/main/java/alluxio/AbstractThriftClient.java

```diff
@@ -117,7 +117,7 @@
         }
         exception = new TException(ae);
       } catch (TException e) {
-        LOG.error(e.getMessage(), e);
+        LOG.warn(e.getMessage(), e);
```

Comment:
are you okay with simply `LOG.warn(e.getMessage())` for consistency with the other cases (to avoid verbose logging on retries)